invenio_cli/cli.py

```diff
@@ -108,7 +108,7 @@ def containerize(pre, force, install_js, stop):
         DockerHelper(cli_config.get_project_shortname(), local=False)
     )
 
-    commands.containerize(pre=pre, force=force, install=install_js, stop=stop)
+    commands.containerize(pre=pre, force=force, install=install_js)
```

Comment:
major: cannot comment on those lines. should remove `stop` argument and click option, since it is not used anymore inside this command?